tests/framework/installer/settings.go

```diff
@@ -46,6 +46,10 @@ func readManifest(provider, filename string) string {
 
 func readManifestFromGithub(rookVersion, provider, filename string) string {
 	url := fmt.Sprintf("https://raw.githubusercontent.com/rook/rook/%s/cluster/examples/kubernetes/%s/%s", rookVersion, provider, filename)
+	return readManifestFromURL(url)
```

Comment:
Is this change fixing anything?

Reply:
A new method `readManifestFromURL()` was added to load from a different url than the rook github.